mkdocs.yaml

```diff
@@ -0,0 +1,117 @@
+site_name: Deeporigin CLI and Python Client
```

Comment:
"Deep Origin" should be two words

Reply:
ok. what are the conventions we want to follow? I can enforce them programatically throughout:

- Deep Origin, when writing in English
- deeporigin, when talking about client, programs, etc. 
- deep-origin, when talking about owners of resources (e.g, application namespaces)?

Reply:
fixed

Reply:
> Deep Origin, when writing in English
+ 1

> deeporigin, when talking about client, programs, etc.
+1 for this client.

> deep-origin, when talking about owners of resources (e.g, application namespaces)?
As Kenny pointed out, we generally use `deeporigin` rather than `deep-origin`. As part of moving our blueprints to a proper database, I think this would be a good point to standardize our DO Deep Origin resource namespace to `deeporigin`.